.github/workflows/python-test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         key: ${{ matrix.os }}-engines
     - name: Test with pytest
       run: |
-        pytest --log-cli-level=10
+        pytest --log-cli-level=10 -s
```

Comment:
What does this do?

Reply:
It disables output capturing by pytest (see [here](https://stackoverflow.com/questions/14405063/how-to-see-normal-print-output-created-during-pytest-run)). I changed it to get more debug information which may have helped with fixing the tests.